tests/integration.rs

```diff
@@ -22,12 +20,34 @@ fn configure() {
 fn db_find_test() {
     configure();
 
-    let mut matches = Index::Name.find(PACKAGE_NAME);
+    let rpm_info = Command::new("rpm")
+        .arg("-q")
+        .arg("rpm-devel")
+        .arg("--queryformat=%{NAME}\n%{EPOCH}\n%{VERSION}\n%{RELEASE}\n%{ARCH}\n%{LICENSE}\n")
```

Comment:
Maybe this ought to be a helper method that just grabs the fields one at a time.  It would be a lot less troublesome.